regression/barebones/go.sum

```diff
@@ -4,26 +4,32 @@ github.com/golang/protobuf v1.2.0 h1:P3YflyNX/ehuJFLhxviNdFxQPkGK5cDcApsge1SqnvM
 github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
 github.com/hpcloud/tail v1.0.0 h1:nfCOvKYfkgYP8hkirhJocXT2+zOD8yUNjXaWfTlyFKI=
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
-github.com/hyperledger/fabric-test v1.1.1-0.20191126160459-3b39434023ee h1:Ba0qI2fqC5CGy/VDyxlBB7enyBlV3teyQddk5XhOK5g=
-github.com/hyperledger/fabric-test v1.1.1-0.20191126160459-3b39434023ee/go.mod h1:7K2xXfkAUybufYc/nYZEfJx/B84HnjHHsy6eKWZJyc4=
+github.com/hyperledger/fabric-test v1.4.4 h1:17Yu+p0QQZlswQFHYs9Jt/2hTgunsqInhMU8qXdJeDc=
```

Comment:
shouldn't this point to master?

Reply:
Corrected, I forgot to add the `replace ` directive